### DISCUSSION TITLE
Welcome to MOSA-Project Discussions!

### DISCUSSION THREAD

**Discussion by tgiphil** [mosa/MOSA-Project#1028] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by AnErrupTion** [comment]
Hello! I'm AnErrupTion, I'm 15 years old and I live in France! I love MOSA and try my best to actively improve it, in any way, shape or form. I hope we can all build a community together!
